docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md

```diff
@@ -22,10 +22,10 @@ When you include asynchronous code in your app, you should consider and possibly
 - [Reviewing and Running the Example App](#BKMD_SettingUpTheExample)
 
 > [!NOTE]
-> To run the example, you must have Visual Studio 2012 or newer and the .NET Framework 4.5 or newer installed on your computer.
+> To run the example, you must have Visual Studio 2012 or newer and .NET Framework 4.5 or newer installed on your computer.
```

Comment:
we should probably create an issue to revisit this article and its samples. It probably works with Core too, it uses msdn links, etc.

Reply:
@mairaw, I opened #16833